source-mongodb/pull.go

```diff
@@ -174,7 +133,13 @@ func (s *captureState) Validate() error {
 
 const (
 	StatusBackfill = 0
-	StatusStreaming = 1
+	// Value of "1" was previously used to designate `StatusStreaming`, however
+	// an update in how we process streams meant that we needed to backfill
+	// bindings again to end up with a clean state, so `StatusStreaming` was
+	// updated to "2", and bindings that still had `Status = 1` in their state
+	// were backfilled. All of this is to say: do not re-use the value 1 in this
+	// enum.
+	StatusStreaming = 2
```

Comment:
Having 2 possible statuses here for expressing "is the backfill done" seems a little smelly to me, especially since I don't think the `StatusBackfill` is being used anywhere. How about a boolean property like `BackfillComplete`? All states going forward would have this initialized as `false` until it is set to `true` by the updated connector, so this would also accomplish the desire to re-backfill all collections.